polygon/rest/base.py

```diff
@@ -74,11 +74,6 @@ def _get(
         raw: bool = False,
         options: Optional[RequestOptionBuilder] = None,
     ) -> Any:
-        if params is None:
-            params = {}
-        params = {str(k): str(v) for k, v in params.items() if v is not None}
-        logger.debug("_get %s params %s", path, params)
-
```

Comment:
Fields, which is taking params can be optional; I feel like this is "debug code" that was never removed; same logic is taken care of In the _get_params function. Seems redundant/dead code.

Reply:
I'm not sure if it's debug code. It seems to be converting every item into a string

Reply:
The tests are fine with this change I'm guessing?

Reply:
Yes, tests are passing. 
values are added to params only if they are not empty. I can’t think of a reason why we would want to build a new dict based on what was provided and ensure that all keys are type str.

Reply:
You're right. This code came before I figured out the introspection for `_get_params`